<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entertaining Chess</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/entertaining-chess.svg</normaloff>:/assets/entertaining-chess.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="MainScreen" name="mainScreen"/>
      <widget class="SettingsScreen" name="settingsPage"/>
      <widget class="OnlineScreen" name="onlineScreen"/>
      <widget class="GameScreen" name="gameScreen"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainScreen</class>
   <extends>QWidget</extends>
   <header>screens/mainscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameScreen</class>
   <extends>QWidget</extends>
   <header>screens/gamescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OnlineScreen</class>
   <extends>QWidget</extends>
   <header>screens/onlinescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsScreen</class>
   <extends>QWidget</extends>
   <header>screens/settingsscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
